statsmodels/tsa/stattools.py

```diff
@@ -1620,3 +1620,279 @@ def _sigma_est_kpss(resids, nobs, lags):
         resids_prod = np.dot(resids[i:], resids[:nobs - i])
         s_hat += 2 * resids_prod * (1. - (i / (lags + 1.)))
     return s_hat / nobs
+
+
+class ur_za(object):
```

Comment:
Any reason not to use standard CamelCase naming?

Reply:
To be perfectly honest I don't recall why I did it this way other than the fact that the the actual call is all lower (i.e. 'za'). More than happy to change it to CC (i.e. class ZivotAndrews).

Reply:
Changed the class name to ZivotAndrewsUnitRoot.